docs/source/getting_started/index.rst

```diff
@@ -80,30 +80,30 @@ A quick try out of the installation can be done with:
 Dependencies
 ~~~~~~~~~~~~~
 
-DPF-Core dependencies are automatically checked installed when the packages is installed.
-Those dependencies are:
+DPF-Core dependencies are automatically checked when packages are 
+installed. The dependencies are:
 
-- ansys.grpc.dpf : package with grpc code generated from protobufs
-- psutil
-- progressbar2
+- `ansys.grpc.dpf` with thegRPC code generated from protobufs
```

Comment:
Check spaces.

Also, these packages should have links to the pypi packages.  Google psutil pypi and select the first hit.